index.js

```diff
@@ -187,6 +187,12 @@ async function fastifyStatic (fastify, opts) {
 
         return reply.callNotFound()
       }
+
+      // err is an object created by `http-errors`
+      if (err.status === 404) {
```

Comment:
If the status is 404 and it is created by `http-errors`. Then it should be good to go with the below line `reply.send(err)`. Isn't it?

Reply:
I see where is the problem. ignore this comment.